app/assets/stylesheets/application.scss

```diff
@@ -14,8 +14,6 @@
  *= require_self
  */
 @import 'bootstrap';
-@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap');
-
 * {
-  font-family: 'Source Sans Pro', sans-serif;
+  font-family: Arial, Helvetica, sans-serif
```

Comment:
- [ ] Please remove this file, it generates an error that multiple files with the same output path cannot be linked